pom.xml

```diff
@@ -475,33 +469,9 @@
     </dependency>
 
     <dependency>
-      <groupId>junit-addons</groupId>
-      <artifactId>junit-addons</artifactId>
-      <version>1.4</version>
-      <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>xerces</groupId>
-          <artifactId>xmlParserAPIs</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>xerces</groupId>
-          <artifactId>xercesImpl</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-
-    <dependency>
-      <groupId>mockobjects</groupId>
-      <artifactId>mockobjects-core</artifactId>
-      <version>0.09</version>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>mockobjects</groupId>
-      <artifactId>mockobjects-jdk1.4-j2ee1.3</artifactId>
-      <version>0.09</version>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>4.6.1</version>

```

Comment:
This gives us two separate mocking frameworks in the tests: Mockito and Easymock. Can you standardize on Mockito and remove Easymock?

Reply:
I think I can do that as separate PR, it's a bit too much work right now, especially since I'm already working on removing assertion messages (as requested in https://issues.apache.org/jira/browse/CONFIGURATION-817).

I tried using EasyMock as replacement of mockobjects first just to prevent a second mocking framework, but as far as I can tell EasyMock requires every call to be specified in order up front, whereas mockobjects and Mockito can be setup in any order (as long as the matchers are used correctly). I got so many errors that I gave up.

Reply:
FYI, I'm already working on replacing EasyMock.